tests/lib/Controller/ConfigControllerTest.php

```diff
@@ -68,6 +64,33 @@ private function getConfigMock($codeVerifier, $clientSecret) {
 	 * @before
 	 */
 	public function setUpMocks(): void {
+		$this->l = $this->createMock(IL10N::class);
+		$this->l->expects($this->any())
+			->method('t')
+			->willReturnCallback(function ($string, $args) {
+				return vsprintf($string, $args);
+			});
+	}
+
+	public function testOauthRedirectSuccess():void {
+		$configMock = $this->getConfigMock(
+			str_repeat("A", 128), str_repeat("S", 50));
+		$configMock
+			->method('setUserValue')
+			->withConsecutive(
+				['testUser', 'integration_openproject', 'token', 'oAuthAccessToken'],
+				['testUser', 'integration_openproject', 'refresh_token', 'oAuthRefreshToken'],
+				['testUser', 'integration_openproject', 'user_id', 1],
+				['testUser', 'integration_openproject', 'user_name', 'Tripatti Himal'],
```

Comment:
```suggestion
				['testUser', 'integration_openproject', 'user_name', 'Tripathi Himal'],
```
It should be `Tripathi` :wink:

Reply:
good catch! changed!